src/main/java/io/cdap/e2e/pages/locators/CdfSysAdminLocators.java

```diff
@@ -51,4 +53,47 @@ public class CdfSysAdminLocators {
 
   @FindBy(how = How.XPATH, using = "//*[@data-cy='Clear All']")
   public static WebElement clearAll;
+
+  @FindBy(how = How.XPATH, using = "//*[@data-cy='wizard-result-icon-close-btn']")
+  public static WebElement clickOnCloseNamespaceWindow;
```

Comment:
Can we remove using click while defining locators , click/sendKeys are user functions which we can use in methods but not in while declaring locators . the naming should be like CloseNamespaceWindowButton .

Reply:
Change done.